cast/tests/e2e/helpers/devnet.rs

```diff
@@ -0,0 +1,47 @@
+use ctor::{ctor, dtor};
+use url::Url;
+use std::process::{Command, Stdio};
+use std::string::ToString;
+use std::thread;
+use std::time::Duration;
+
+pub const URL: &str = "http://127.0.0.1:5055/rpc";
+pub const NETWORK: &str = "testnet";
+const SEED: u32 = 1_053_545_548;
+
+#[cfg(test)]
+#[ctor]
+fn start_devnet() {
+    let port = Url::parse(URL).unwrap().port().unwrap_or(80).to_string();
```

Comment:
We could try to generate a random port to avoid errors when 5055 is taken

Reply:
I don't think this is good idea, we could have picked some restricted port. 5055 should be rather safe, and if something blocks it the tests will fail with something like "port is taken"